src/index.html

```diff
@@ -65,5 +70,214 @@ <h1 class="headline__title">
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <ul class="project-card__tags-list">
+            <li class="project-card__tags-list-item">html</li>
+            <li class="project-card__tags-list-item">css</li>
+            <li class="project-card__tags-list-item">javascript</li>
+          </ul>
+          <div class="project-card__actions">
+            <button class="button" type="button">See Project</button>
+          </div>
+        </div>
+      </div>
+      <div class="project-card">
+        <img
+          class="project-card__image"
+          src="./assets/images/snapshoot-4.jpg"
+          alt="Snapshoot 1"
+        />
+        <div class="project-card__body">
+          <div class="project-card__header">
+            <h2 class="project-card__header-title">Multi-Post Stories</h2>
+            <div class="project-card__header-subtitle">
+              <span>CANOPY</span>
+              <span>Back End Dev</span>
+              <span>2015</span>
+            </div>
+          </div>
+          <p class="project-card__description">
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <ul class="project-card__tags-list">
+            <li class="project-card__tags-list-item">html</li>
+            <li class="project-card__tags-list-item">css</li>
+            <li class="project-card__tags-list-item">javascript</li>
+          </ul>
+          <div class="project-card__actions">
+            <button class="button" type="button">See Project</button>
+          </div>
+        </div>
+      </div>
+    </section>
+    <section class="about">
+      <div class="about-me">
+        <div class="about-me__body">
+          <div class="about-me__title">
+            <h2>
+              About <br />
+              Myself
+            </h2>
+          </div>
+          <div class="about-me__text">
+            <p>
+              Hello I’m a software developer! I can help you build a product ,
+              feature or website Look through some of my work and experience! If
+              you like what you see and have a project you need coded, don’t
+              hestiate to contact me.
+            </p>
+          </div>
+          <div class="about-me__actions">
+            <a class="link" href="#">LET’S CONNECT</a>
+            <ul class="social-stack social-stack--no-gap">
+              <li>
+                <a class="icon-botton">
+                  <img src="./assets/icons/github.svg" alt="" />
+                </a>
+              </li>
+              <li>
+                <a class="icon-botton">
+                  <img src="./assets/icons/linkedin.svg" alt="" />
+                </a>
+              </li>
+              <li>
+                <a class="icon-botton">
+                  <img src="./assets/icons/twitter.svg" alt="" />
+                </a>
```

Comment:
- [ ] What do you think about adding the AngelList and Medium icons here as well? I know you might not be having accounts for them right now but in the near future, Microverse will be helping you create accounts for both of them and they're both popular platforms. It's better to have the icons right now so you would not have to change your Portfolio in the near future :)

Reply:
### Thank you for your comment.
issue fixed here [Add AngelList and Medium icons](https://github.com/teckim/portfolio/pull/2/commits/2e27d9eebe1fa42076b7340ddd3f25ec4d1a508c)